src/main/java/hudson/plugins/sonar/MsBuildSQRunnerBegin.java

```diff
@@ -67,13 +69,20 @@
    * @since 2.4.3
    */
   private String msBuildScannerInstallationName;
-  private final String sonarInstallationName;
-  private final String projectKey;
-  private final String projectName;
-  private final String projectVersion;
-  private final String additionalArguments;
+  private String sonarInstallationName;
+  private String projectKey;
+  private String projectName;
+  private String projectVersion;
+  private String additionalArguments;
 
   @DataBoundConstructor
+  public MsBuildSQRunnerBegin() {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1186)